lib/datasource/metadata.js

```diff
@@ -100,4 +105,6 @@ function addMetaData(dep, datasource, lookupName) {
       delete dep[url];
     }
   }
+  /* eslint-disable consistent-return */
```

Comment:
Why is this needed?

Reply:
It was needed because I added a return statement there which was not there earlier, 
`(return dep;)` . Lint started throwing a no return statement error for that function. I don't know why that function should not have a return statement?

Reply:
I needed that return statement so that I could get the data back from the addMetaData() in the test. It doesn't break anything.